scripts/limonade/src/index.ts

```diff
@@ -138,6 +177,36 @@ async function run() {
     }
   };
 
+  const isOffMarket = (order: { state: Order; address: PublicKey }) => {
+    const priceInputMint = priceStore[order.state.inputMint.toString()];
+    const priceOutputMint = priceStore[order.state.outputMint.toString()];
+
+    if (!priceInputMint || !priceOutputMint) {
+      // If we don't have price info, we will not consider it off-market
+      return false;
+    } else {
+      const inputAmount = order.state.remainingInputAmount;
+      const outputAmount = order.state.expectedOutputAmount.sub(
+        order.state.filledOutputAmount
+      );
+
+      const ratio =
```

Comment:
nit, this ratio is probably going to be recomputed many times for a given pair, maybe it makes sense to cache? for now it's probably not going to be too big a deal, so will leave it up to you

Reply:
seems too early to optimize this